source/_includes/common-tasks/enable_i2c.md

```diff
@@ -62,3 +62,28 @@ You can enable i2c via this terminal:
   sync
   reboot
   ```
+#### Check I2C activation on OS level
+
+After the reboot of the host, the I2C activation should now be visible under /dev. You may check by using the add-on `Terminal & SSH` and list the directory `/dev`. If `ls` shows an entry `i2c-1` or similar, your I2C config might work. You can check the status of I2C kernel modules by entering `lsmod | grep i2c`. If they are loaded, you should find at least the entry `i2c_dev`. Active usage of the modules is indicated by a number, e.g. `i2c_dev 20480 2` would indicate two active I2C data sources (e.g. a BMP280 temperature & pressure sensor) 
+
+An active I2C can also be check with a multi meter showing 3.3 V on the I2C pins GPIO2 and GPIO3. 
+
+#### Configuration of the Home Assistant to enable a BME280 sensor 
+  
+Connect a I2C device to the GPIO2 (SDA1), GPIO3 (SCL) and VCC/GND as needed. Ensure that this wiring is working (e.g by using a Raspberry OS with configured I2C) and note the I2C address of the  device. For a BMP280 this might be 0x76.  
+
+Edit the `configuration.yaml` e.g. via the add-on `File Editor`. You need to activate the  binary sensor plattform `raspihats` to enable I2C for the Home Assistant core. The I2C sensor needs and additional entry. A working entry may look like this:
+
+```shell
+# Enable Raspihats
+binary_sensor:
+  - platform: raspihats
+
+# Enable BMP 280 sensor
+sensor:
+  - platform: bmp280
+    i2c_address: 0x76
+```
+
+After a reboot of host, you can find the sensor listed in `Developer Tools` in the tab `States`as `sensor.bmp280_temperature`and `sensor.bmp280_pressure`. 
+  
```

Comment:
This is not related to this page. Please remove that. This belongs on a integration page. Not in the common tasks.

Reply:
Removed the section as requested